planner_id: "RRTConnectkConfigDefault"
#planner_id: "RRTStarkConfigDefault"
planning_attempts: "5"
planning_time: "8"
tolerance_pose: "0.01"
tolerance_angle: "0.01"
babbling_group: "both_arms"
babbling_arm: "left_arm"
secondary_arm: "right_arm"
approach_radius: "0.12"
